preprocess-kbs.scm

```diff
@@ -101,15 +101,15 @@
 (ure-logger-set-filename! log-filename)
 
 ;; Load KBs to reason on
-(define db-lst (load-kbs (list "kbs/gene-level-dataset_2020-04-21/GO_2020-04-01.scm"
-                               "kbs/gene-level-dataset_2020-04-21/GO_annotation_gene-level_2020-04-01.scm")
+(define db-lst (load-kbs (list "kbs/GO_2020-07-21.scm"
+                               "kbs/GO_annotation_gene-level_2020-07-21.scm")
                          #:subsmp ss
                          #:filter-out (lambda (x)
                                         (or (GO_term? x)
                                             (inheritance-GO_term? x)))))
 
 ;; Helpers
-(define ConceptT (Type "ConceptNode"))
+(define ConceptT (TypeInh "ConceptNode"))
```

Comment:
I see, that might be where the problem is, even though obviously you don't have a choice here. So let me try and debug that locally.

Reply:
Never mind my comment, hopefully this works fine.